js/catalog-search.js

```diff
@@ -4,23 +4,25 @@ bcpl.catalogSearch = (($, queryStringer, waitForExistence, constants) => {
 	const catalogSearchSelector = '#catalog-search, .catalog-search';
 	const resultsInfoContainerSelector = '.gsc-above-wrapper-area-container';
 	const searchCatalogButton = '<td><button id="catalog-search" class="btn btn-primary pull-right">Search the Catalog</button></td>';
-
+	const showResults = "#resInfo-0:contains('About')";
```

Comment:
so a couple things

1. Looks like `showResults` is a selector so the varaible name should be `showResultsSelector`. We have actually started moving to a pattern more recenlty of having a selectors object and then this would be fine

```js
const selectors = {
  showResults: 'results seelctor goes here'
};
```

but it's not worth the refactor here.

2. This selectors seems a little off to me. `#` indicates an id, so there should only be one instance of that id on the page. Is that not true? meaning I would expect this selector to work.:

```js
const showResultsSelector = '#resInfo-0';
```
However, this all my be irrelevant.

Reply:
Variable name changed to reflect Selector type